### ISSUE TITLE
No torrents

### ISSUE THREAD

**Issue by VatraXD** [teal77/trireme#74]
Hi!
The app looks great, but I don't see my torrents running. I can add them and they show up on the desktop client, but not on the phone.
The download and upload speed also work. Plus in the filtering menu I do see my trackers, so its kinda working? The torrents just won't show and it says no torrents. Maybe because my server is on version 1.3.15? Thanks!

**Comment by teal77** [comment]
Yeah I didn't test deluge 1 at all with the recent version. Did the
previous version work?

**Comment by VatraXD** [comment]
I'm a new user, I don't know. Give me a few moments and I'll check. Weird that it connects but torrents wont show.

**Comment by VatraXD** [comment]
Yes, with app version 1.3.0 it shows the torrents.

**Comment by nini1294** [comment]
I am having a similar issue, and not sure if it is related. The logs that I see on the client side when this happens are below. Adding if that helps.

```
[ERROR   ] 05:30:25 rpcserver:304 Exception calling RPC request: 'int' object has no attribute '__getitem__'
Traceback (most recent call last):
File "/usr/lib/python2.7/dist-packages/deluge/core/rpcserver.py", line 291, in dispatch
     auth_level = self.factory.authorized_sessions[self.transport.sessionno][0]
     TypeError: 'int' object has no attribute '__getitem__'
```

App version: 1.4.1
Deluge version: 1.3.2

**Comment by teal77** [comment]
Version 1.4.3 released with fix for Deluge 1 connection > empty list on open.
